src/commands/heroku.rs

```diff
@@ -12,25 +13,90 @@ use crate::config::Config;
 struct HerokuApp {
     id: String,
     name: String,
+    released_at: String,
+    web_url: String,
+}
+
+// Get app by name or id
+#[command]
+pub fn get_app(ctx: &mut Context, msg: &Message, mut args: Args) -> CommandResult {
+    let config = bot_config(ctx);
+
+    let app_name = args
+        .single::<String>()
+        .expect("You must include an app name");
```

Comment:
`expect()` also panics on `None`. How does error handling in Serenity work? Can we return some error to indicate failure?

From a quick look at the docs, I noticed there's a `#[num_args(1)]` attribute that can be used to mark the command as requiring exactly one arg. Maybe this is the easiest option in this case?

Reply:
Good point! I added that in, along with some handling for the error in src/lib.rs so that it notifies the Discord user in the channel they tried to send the command in.